src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const keys = [];
+
+  for (const key in robot) {
+    keys.push(robot[key]);
+  }
+
+  for (let i = 0; i < keys.length; i++) {
+    if (keys.includes(keys[i], i + 1)) {
+      return null;
+    }
+  }
```

Comment:
Instead of adding extra logic to find repeated values, you can do this right in the loop where you inverse, just add a check if `inverse` object already has such property

Reply:
Okay, I know how to fix that